### PR TITLE
Use prefetched ProjectLocales on team/project dashboards

### PR DESCRIPTION
**Pull request by mathjazz**
It should speed up Project and Team dashboard default tabs by some 20 %.

@jotes r?

### CODE REVIEW

File: pontoon/base/models.py [hunk 1]
```diff
@@ -1169,11 +1169,9 @@ def get_latest_activity(cls, self, extra=None):
         """
         latest_translation = None
 
-        if hasattr(self, 'fetched_latest_translation'):
-            if self.fetched_latest_translation:
-                latest_translation = self.fetched_latest_translation[0].latest_translation
-            else:
-                latest_translation = None
+        if hasattr(self, 'fetched_project_locale'):
```
Comment (jotes) [inline pontoon/base/models.py]: `if getattr(self, 'fetched_project_locale'):` and remove the second if?

File: pontoon/teams/templates/teams/includes/projects.html [hunk 2]
```diff
@@ -16,12 +16,14 @@
 
   {{ ProjectList.header(request=True) }}
 
+  {% set locale_projects = locale.available_projects_list() %}
+
```
Comment (mathjazz) [inline pontoon/teams/templates/teams/includes/projects.html]: We already make this request in the main template, so we could also move the logic for showing and hiding enabled project on first load to JS. But, that wouldn't have any noticable perf impact. And would be a much bigger codebase change. So I'm keeping it as is for now.

File: pontoon/base/models.py (hunk 1 above)
Comment (mathjazz) [inline pontoon/base/models.py]: Thanks!

I ended up using this: `if getattr(self, 'fetched_project_locale', None):`

**Comment by mathjazz** [comment]
@jotes r?

I found another (bigger) optimization for the Team and Project dashboard. Check out the second commit. Together, both commits should bring around 70% faster loading [Team](localhost:8000/pl/) and [Project](localhost:8000/projects/pontoon-intro/) dashboards.

**Comment by jotes** [comment]
R+, looks good

**Pull request merged**